include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -114,7 +119,8 @@ typedef struct _oe_sgx_enclave_properties
     EXTENDED_PRODUCT_ID,                                                  \
     FAMILY_ID,                                                            \
     ALLOW_DEBUG,                                                          \
```

Comment:
It looks like this code doesn't follow [OE style guidance](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md) which says:

> Use lower_snake_case for variable, member/field, and function names.

Reply:
It turns out changing the argument names to lower_snake_case that are the same as the name of any struct members lead to compilation errors.

For example, the following would fail
```
#define _OE_SET_ENCLAVE_SGX_IMPL(                                         \
    product_id,                                                           \
    ...
```
Unfortunately, most of the upper_snake_case arguments here match the name of struct members.  I'd prefer leave it unchanged for now. Any comments? @dthaler

Reply:
Ok to file a separate issue to track and deal with it in a future PR

Reply:
https://github.com/openenclave/openenclave/issues/3998

Reply:
to answer the question, you just have to use names that don't collide, like prefix with underscore, or add some suffix